hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamer.java

```diff
@@ -1802,6 +1802,44 @@ private void testParquetDFSSource(boolean useSchemaProvider, List<String> transf
     testNum++;
   }
 
+  @Test
+  public void testMultipleTransformers() throws Exception {
```

Comment:
lets use this opp to pull a new Transformer test?

Reply:
Addressed in https://github.com/apache/hudi/pull/8514